apple/Handlers/RNHoverHandler.m

```diff
@@ -163,17 +164,58 @@ - (RNGestureHandlerEventExtraData *)eventExtraData:(UIGestureRecognizer *)recogn
 
 #else
 
-@implementation RNHoverGestureHandler
+@implementation RNHoverGestureHandler {
+  NSTrackingArea *trackingArea;
+  RNGHUIView *_view;
+}
 
 - (instancetype)initWithTag:(NSNumber *)tag
 {
-  RCTLogWarn(@"HoverGestureHandler is not supported on macOS");
   if ((self = [super initWithTag:tag])) {
     _recognizer = [NSGestureRecognizer alloc];
   }
+
   return self;
 }
 
+- (void)bindToView:(RNGHUIView *)view
+{
+  _view = view;
+
+  NSTrackingAreaOptions options =
+      NSTrackingMouseEnteredAndExited | NSTrackingActiveInActiveApp | NSTrackingInVisibleRect;
+
+  trackingArea = [[NSTrackingArea alloc] initWithRect:_view.bounds options:options owner:self userInfo:nil];
+  [_view addTrackingArea:trackingArea];
+}
+
+- (void)unbindFromView
+{
+  [super unbindFromView];
```

Comment:
I assume there's no `super` call in bindToView due to a recognizer not being used. If that's correct, then why do we call `super` here?

I guess the question still stands if the assumption is wrong 😅.

Reply:
Good point, I've added it automatically, without noticing that it requires recognizer.

Removed in [fd79077](https://github.com/software-mansion/react-native-gesture-handler/pull/3027/commits/fd79077dcd98fe0a78392c4cc7fdae66833bf9fe)